docs/azmcp-commands.md

```diff
@@ -354,6 +354,75 @@ azmcp postgres server setparam --subscription <subscription> \
                                --value <value>
 ```
 
+### Azure Database for MySQL Operations
```

Comment:
Since this is a brand new area, please also add the entry to .github/CODEOWNERS following the pattern established by the other services.  There's a link in CONTRIBUTING.md with details

Reply:
CODEOWNERS has src before Areas. But actual folder structure does not have any src folder Example : /src/Areas/Postgres/